nixos/modules/config/nix.nix

```diff
@@ -385,6 +387,7 @@ in
       trusted-public-keys = [ "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=" ];
       trusted-users = [ "root" ];
       substituters = mkAfter [ "https://cache.nixos.org/" ];
+      system-features = defaultSystemFeatures;
```

Comment:
The default value is now the initial value, which should be documented in the description field.

Reply:
It was already documented before?
```
  defaultText = literalExpression ''[ "nixos-test" "benchmark" "big-parallel" "kvm" "gccarch-<arch>" ]'';
```